index.html

```diff
@@ -30,6 +30,10 @@ <h2>Instructions:</h2>
                         <label for="numLynx">How many Lynx?</label>
                         <input type="number" class="inputBox" id="numLynx" name="numLynx" min="1" max="50" required>
                     </div>
+                    <div class="userInput">
+                        <label for="preLives">You will have this many lives:</label>
```

Comment:
Maybe a shorter way of saying this. Possibly 'Number of lives: '

Reply:
Revisit the instructions to introduce this idea of lives lost. Line 20 'Click on the squares to find all the lynx in the forest... but beware of the snakes! You will lose a life each time you get a bite'